src/main/java/com/algorand/algosdk/transaction/SignedTransaction.java

```diff
@@ -65,6 +68,11 @@ public SignedTransaction(Transaction tx, LogicsigSignature lSig, String txId) {
     private SignedTransaction() {
     }
 
+    public SignedTransaction authAddr(Address authAddr) {
```

Comment:
Is there a way to make this private? At the very minimum we'll need a javadoc comment here stating that it should not be set manually.

Reply:
No, Account calls it while signing. I'll add a comment.